app/views/posts/show.html.erb

```diff
@@ -1,2 +1,21 @@
-<p>Details of one post will be displayed here</p>
-<p>For posts controller testing</p>
\ No newline at end of file
+<div class="card" style="width: 30rem;">
+  <ul class="list-group list-group-flush" id="all-posts"> 
+      <li class="list-group-item">
+        <h5><strong><%= @post.title %></strong> by <%= @user.name %></h5>
+        <div class="d-flex flex-row statistics">            
+          <p>Comments: <%= @post.comments_counter %></p>
+          <p>Likes: <%= @post.likes_counter %></p>
+        </div>
+      </li>
+      <li class="list-group-item"><%= @post.text %></li>
+      <li>
+        <p><strong>Comments:</strong></p>
+        <% @post.most_recent_five_comments.each do |c| %>
+          <p><strong><%= c.user.name %></strong>: <%= c.text %></p>
+        <% end %>
```

Comment:
- [ ] Kindly ensure that all the comments are listed in the post_show_page as illustrated in the wireframe. ℹ️ 

<img width="167" alt="CAROL" src="https://github.com/Mesi21/visionOfLife/assets/103386183/94759263-0295-4345-8f2a-f991fa934b3f">